addons/website_sale/static/src/js/checkout.js

```diff
@@ -59,23 +59,50 @@ publicWidget.registry.websiteSaleCheckout = publicWidget.Widget.extend({
      * @return {void}
      */
     async _changeAddress(ev, rowAddrClass, cardClass) {
+        // Find the old card that is currently selected
         const oldCard = document.querySelector(
             `.${rowAddrClass} .card.border.border-primary`
         );
-        oldCard.classList.add(cardClass);
-        oldCard.classList.remove('bg-primary', 'border', 'border-primary');
+        if (oldCard) {
```

Comment:
The code in this function is repeated too many times. I suggest creating a method like this:

```javascript
toggleCardButtons(card, show) {
    const deleteButton = card.querySelector('#delete-button');
    const defaultButton = card.querySelector('#default-button');
    if (deleteButton) {
        deleteButton.style.display = show ? 'inline-block' : 'none';
    }
    if (defaultButton) {
        defaultButton.style.display = show ? 'inline-block' : 'none';
    }
}
```

This method will be called when clicking on a button to show or hide the buttons as needed. Hope you understand.

Reply:
Hello @kpt-odoo ,
Changes are done.

thanks for suggestions. :slightly_smiling_face: